packages/react-composites/src/composites/CallComposite/adapter/AzureCommunicationCallAdapter.ts

```diff
@@ -2023,5 +2018,5 @@ export const createTeamsCallAdapterFromClient = async (
 };
 
 const isCallError = (e: Error): e is CallError => {
-  return e['target'] !== undefined && e['innerError'] !== undefined;
+  return (e as CallError)['target'] !== undefined && (e as CallError)['innerError'] !== undefined;
```

Comment:
nit: to avoid the `as` can you do as you did above and use `in`?
```suggestion
  return 'target' in e && 'innerError' in e;
```